[2024-02-27T10:46:42.120+0700] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: missed_deadline_job_dag.spark_transform_missed_deadline_job scheduled__2024-02-26T00:00:00+00:00 [queued]>
[2024-02-27T10:46:42.127+0700] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: missed_deadline_job_dag.spark_transform_missed_deadline_job scheduled__2024-02-26T00:00:00+00:00 [queued]>
[2024-02-27T10:46:42.127+0700] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-27T10:46:42.141+0700] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): spark_transform_missed_deadline_job> on 2024-02-26 00:00:00+00:00
[2024-02-27T10:46:42.142+0700] {standard_task_runner.py:60} INFO - Started process 14864 to run task
[2024-02-27T10:46:42.145+0700] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'missed_deadline_job_dag', 'spark_transform_missed_deadline_job', 'scheduled__2024-02-26T00:00:00+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/missed_deadlinee_dag.py', '--cfg-path', '/tmp/tmpddzmrsr2']
[2024-02-27T10:46:42.147+0700] {standard_task_runner.py:88} INFO - Job 104: Subtask spark_transform_missed_deadline_job
[2024-02-27T10:46:42.214+0700] {task_command.py:423} INFO - Running <TaskInstance: missed_deadline_job_dag.spark_transform_missed_deadline_job scheduled__2024-02-26T00:00:00+00:00 [running]> on host MinhNhut.
[2024-02-27T10:46:42.299+0700] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='missed_deadline_job_dag' AIRFLOW_CTX_TASK_ID='spark_transform_missed_deadline_job' AIRFLOW_CTX_EXECUTION_DATE='2024-02-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-26T00:00:00+00:00'
[2024-02-27T10:46:42.300+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-27T10:46:42.301+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/nhut/etl-***/pipeline/spark_missed_deadline_job.py']
[2024-02-27T10:46:42.305+0700] {subprocess.py:86} INFO - Output:
[2024-02-27T10:46:42.306+0700] {subprocess.py:93} INFO - /usr/bin/bash: line 1: /home/nhut/etl-***/pipeline/spark_missed_deadline_job.py: Permission denied
[2024-02-27T10:46:42.307+0700] {subprocess.py:97} INFO - Command exited with return code 126
[2024-02-27T10:46:42.321+0700] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/nhut/etl-airflow/.venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/nhut/etl-airflow/.venv/lib/python3.10/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 126.
[2024-02-27T10:46:42.323+0700] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=missed_deadline_job_dag, task_id=spark_transform_missed_deadline_job, execution_date=20240226T000000, start_date=20240227T034642, end_date=20240227T034642
[2024-02-27T10:46:42.335+0700] {standard_task_runner.py:107} ERROR - Failed to execute job 104 for task spark_transform_missed_deadline_job (Bash command failed. The command returned a non-zero exit code 126.; 14864)
[2024-02-27T10:46:42.358+0700] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-27T10:46:42.377+0700] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
